spec/system/page_objects/components/theme_translation_text_area.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Components
+    class ThemeTranslationTextArea < PageObjects::Components::Base
+      def type_input(content)
+        editor_input.send_keys(content)
+        self
+      end
+
+      def fill_input(content)
+        # Clear the input before filling it in because capybara's fill_in method doesn't seem to replace existing content
+        # unless the content is a blank string.
```

Comment:
We use `fill_in` in several other tests, it should be working fine. I suspect you're running into other issues here.